charts/metal-control-plane/Chart.yaml

```diff
@@ -1,6 +1,11 @@
 ---
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for deploying the metal control plane in K8s
 name: metal-control-plane
-version: 0.3.17
\ No newline at end of file
+version: 0.3.18
+dependencies:
+  - name: meilisearch
+    version: ~0.1.44
+    repository: https://meilisearch.github.io/meilisearch-kubernetes
+    condition: auditing.enabled
```

Comment:
Dependency can be dropped, instead move meilisearch deployment role from metalstack cloud to metal-roles.